lib/src/rpc/dial.dart

```diff
@@ -272,7 +333,13 @@ Future<ClientChannelBase> _dialWebRtc(String address, DialOptions options, Strin
         if (uuid != null) {
           callUpdateRequest.uuid = uuid!;
         }
+        final Stopwatch stopwatch = Stopwatch()..start();
+        final currUpdateCall = updateCalls++;
```

Comment:
While this callback does run asynchronously, there's no concept of "atomic"s in dart, as everything is single-threaded (?). So, `updateCalls` need only be an `int`, and I store its value in the local `currUpdateCall` to track the update between "making call update" and "call update $currUpdateCall took".